packages/compass-schema/src/components/coordinates-minichart/coordinates-minichart.jsx

```diff
@@ -328,5 +333,10 @@ CoordinatesMinichart.propTypes = {
   onQueryChanged: PropTypes.func,
 };
 
-export default CoordinatesMinichart;
-export { CoordinatesMinichart };
+const ConnectedCoordinatesMinichart = connect(() => ({}), {
```

Comment:
Super nit, null is also a valid argument here, less code running

```suggestion
const ConnectedCoordinatesMinichart = connect(null, {
```